### ISSUE TITLE
Creacion de Nuget

### ISSUE THREAD

**Issue by janmbaco** [topelab/Core.Resolver#10]
Lo ideal para este proyecto es la creación de un Nugets que se pudiera añadir directamente al punto de entrada de la aplicación.

**Comment by topelab** [comment]
Sí, la idea es que en el punto de entrada de la aplicación elijas la implementación deseada, actualmente cualquiera de las 3:

- Topelab.Core.Resolver.Microsoft
- Topelab.Core.Resolver.Autofac
- Topelab.Core.Resolver.Unity

Los interfaces están definidos en Topelab.Core.Resolver, y es la única necesaria en los proyectos secundarios.
